Library/Basic Set/Basic Set Advantages.adq

```diff
@@ -25189,6 +25189,14 @@
 			"levels": "1",
 			"points_per_level": 5,
 			"reference": "B87",
+			"features": [
+				{
+					"type": "reaction_bonus",
+					"amount": 1,
+					"per_level": true,
+					"situation": "Your class, race, sex (or other group) must be obvious to anyone who meets you; +1 bonus per level, up to +4."
```

Comment:
Don't really need the explanatory text about "+1 bonus per level" etc, as that's part of the automation

Reply:
May also want to put a substitutable marker around the part where it says "class, race, sex (or other group)" so that users are prompted to pick the appropriate thing